cmd/wallet-sdk-gomobile/api/diddocresolution.go

```diff
@@ -34,3 +34,22 @@ func (d *DIDDocResolution) ID() (string, error) {
 
 	return didDocResolutionParsed.DIDDocument.ID, nil
 }
+
+// AssertionMethodKeyID returns the id of key that used for assertion verification.
```

Comment:
Missing a few words: should be `AssertionMethodKeyID returns the id of the key that was used for assertion verification.`